internal/configs/transportserver.go

```diff
@@ -16,6 +17,25 @@ type TransportServerEx struct {
 	PodsByIP        map[string]string
 }
 
+// TransportServerConfigurator generates a transportServer configuration
+type transportServerConfigurator struct {
+	isPlus         bool
+	enableSnippets bool
+	warnings       Warnings
+}
+
+// newTransportServerConfigurator creates a new TransportServerConfigurator
+func newTransportServerConfigurator(
+	isPlus bool,
+	staticParams *StaticConfigParams,
```

Comment:
because we use only one parameter of `*StaticConfigParams`, perhaps it doesn't make sense to pass `*StaticConfigParams`? but instead, just a boolean flag?

Reply:
Done